src/Aspire.Dashboard/Components/Pages/Resources.razor.cs

```diff
@@ -213,9 +210,9 @@ protected override async Task OnInitializedAsync()
 
         async Task SubscribeResourcesAsync()
         {
-            var preselectedVisibleResourceTypes = VisibleTypes?.Split(',').ToHashSet();
-            var preselectedVisibleResourceStates = VisibleStates?.Split(',').ToHashSet();
-            var preselectedVisibleResourceHealthStates = VisibleHealthStates?.Split(',').ToHashSet();
+            var preselectedHiddenResourceTypes = HiddenTypes?.Split(' ').Select(StringUtils.Unescape).ToHashSet();
```

Comment:
What happens if a value (type, state, health state) contains a space?

Reply:
> What happens if a value (type, state, health state) contains a space?

The space is serialized to a +. Then, when appending to the URL, `QueryHelpers.AddQueryString` will escape the + to %2B. Reverse process happens when reading from the query param.

> Why did you change from comma to space?

I noticed that this was the only instance where a space was not being used to separate separate filters. In telemetry filters, space is used in the URL.